config-templates/windows--local-dev-segue-config.properties

```diff
@@ -92,3 +92,6 @@ TWITTER_CALLBACK_URI=http://localhost:8000/auth/twitter/callback
 
 # IPAddress GeoCoding
 IP_INFO_DB_API_KEY=[enter_key_here]
+
+# User Preferences
+SUBJECT_INTEREST=PHYSICS_UNI,PHYSICS_ALEVEL,PHYSICS_GCSE,CHEMISTRY_UNI,CHEMISTRY_ALEVEL,CHEMISTRY_GCSE,MATHS_UNI,MATHS_ALEVEL,MATHS_GCSE

```

Comment:
I wonder if it would be sensible to do the same thing as above with the other user preferences e.g. emails? We could take a hybrid approach of listing email preferences in a config file and hard coding the mandatory ones e.g. SYSTEM.

Just a suggestion, but I realise that the email stuff is still a bit horrid and probably not a massive priority to fix.